package.json

```diff
@@ -2,61 +2,66 @@
   "name": "ticker-admin",
   "version": "1.0.0",
   "license": "GPL-3.0-or-later",
-  "dependencies": {},
+  "dependencies": {
+    "leaflet": "^1.5.1",
+    "leaflet-draw": "^1.0.4",
+    "react-leaflet": "^2.3.0",
+    "react-leaflet-draw": "^0.19.0"
+  },
   "devDependencies": {
```

Comment:
@0x46616c6b We wonder which dependencies are supposed to be assigned as devDependencies

Reply:
My understanding is, that for building react apps it don't need to split dependencies between prod and dev. I assume that only imports from the application will bundle in the production javascript.